subsys/logging/log_backend_adsp.c

```diff
@@ -20,12 +22,10 @@ static int char_out(uint8_t *data, size_t length, void *ctx)
 	return length;
 }
 
-/* Trace output goes in 64 byte chunks with a 4-byte header, no point
- * in buffering more than 60 bytes at a time
- */
-static uint8_t log_buf[60];
+/* No reason to buffer more, trace out is a buffer */
+static uint8_t log_buf[80];
```

Comment:
previously it was 60. Any reason for this change?

Reply:
I've added a comment and define, also using sizeof where appropriate